src/main/java/gyro/google/compute/ComputeInstanceGroupManagerUpdatePolicy.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.Optional;
+
+import com.google.api.services.compute.model.InstanceGroupManagerUpdatePolicy;
+import gyro.core.resource.Diffable;
+import gyro.core.validation.ValidStrings;
+import gyro.google.Copyable;
+
+public class ComputeInstanceGroupManagerUpdatePolicy extends Diffable
+    implements Copyable<InstanceGroupManagerUpdatePolicy> {
+
+    // instanceRedistributionType is not available from Google SDK yet.
+    //    private String instanceRedistributionType;
```

Comment:
As it is not supported so should be removed.

Reply:
This is to make clear that why the field wasn't implemented.